gems/faker/2.23/faker.rbs

```diff
@@ -0,0 +1,33 @@
+# Write the type definition here!
+
+module Faker
+  class Base
+    def self.unique: () -> UniqueGenerator
+  end
+
+  class UniqueGenerator
+    def self.clear: () -> {}
+    def clear: () -> {}
+    def exclude: (Symbol, Array[String]|Array[Hash[Symbol, untyped]], Array[String]) -> Array[String]
```

Comment:
I guess `void` is more appropriate for the returned value. This method doesn't expect the user to use the returned value.

Reply:
I run the sample codes. and it returns Array[String].
But the user expects a void like your suggestion.

I prefer to define Array[String]. please give your opinion. I follow you.

```ruby
Faker::Lorem.unique.exclude :string, [number: 6], %w[azerty wxcvbn]
=> ["azerty", "wxcvbn"]
```

Reply:
Yeah, this method actually returns an array of string. 

The returned value is a returned value of `each` method.
https://github.com/faker-ruby/faker/blob/07234831e201c853c65120e0569d9a810f18586d/lib/helpers/unique_generator.rb#L53-L58
In many cases, the returned value of `each` method is ignored. I think this case also ignores the returned value.

If we define it as `Array[String]` instead of `void`, someone may use the value as an array. But Faker doesn't expect this usage. So I think we should avoid the wrong usage by typing it as `void`.